components/automate-cli/cmd/chef-automate/automateConfigUtils.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import ptoml "github.com/pelletier/go-toml"
+
+func getModeFromConfig(configPath []string) string {
+	initConfigHAPath := initConfigHAPathFlags.path
+	if len(configPath) > 0 {
```

Comment:
if we are only going to read 1st value in the []string, then why are we getting []string in the function argument.
Would it not be better if in the `runDeployCmd` and `runProvisionInfraCmd` we check the no. of values and pass the first value directly.